### DISCUSSION TITLE
Searching for submissions in CP yields inconsistent results

### DISCUSSION THREAD

**Discussion by superextinct** [solspace/craft-freeform#810] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
Using the search bar at `admin/freeform/submissions` does not work as expected, as some submissions are returned by searching for a specific term, while other submissions are not returned when searching for their title. This happens regardless of form or specified start and end date. This appears to happen more towards older submissions (perhaps this has to do with the introduction of separate data tables per form?)

**Steps to reproduce**
![Bildschirmaufnahme 2023-08-07 um 15 26 27](https://github.com/solspace/craft-freeform/assets/41620375/b17d4aa2-7376-47a1-967c-c4906dc53b77)

Submissions of this form have their title set to submission date. Searching for '2023' should yield 12 results, but only 10 items are returned. Searching for '2022' should show one result, but shows none.

**Expected behavior**
The provided search term should be applied to all visible columns and show matching submissions.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.4.16.1
- Freeform Version: 4.1.6
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**

**Comment by justinholtweb** [comment]
I can also confirm this is happening on one of our installs that has forms with a lot of fields and 10k+ submissions.

**Comment by kjmartens** [comment, marked as answer]
I'm sorry for the delay @superextinct.

Unfortunately this is a current limitation in Freeform. We plan to address it in the not-too-distant future, however.

As for the date filter on the charts, this is only supposed to limit the data range in the chart, not the submission results further below. 🙂
